packages/firebase_performance/firebase_performance_web/analysis_options.yaml

```diff
@@ -0,0 +1,4 @@
+include: package:flutter_lints/flutter.yaml
```

Comment:
I don' think this file is necessary. We also use our own `analysis_options.yaml`. [example](https://github.com/FirebaseExtended/flutterfire/blob/1f8d79a3ee2ab6e5a73cd57d1ce164bccfe14879/packages/firebase_ml_custom/example/analysis_options.yaml#L5)